remsfal-service/src/test/java/de/remsfal/service/boundary/project/GarageResourceTest.java

```diff
@@ -0,0 +1,312 @@
+package de.remsfal.service.boundary.project;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.ContentType;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import de.remsfal.service.TestData;
+
+import java.time.Duration;
+
+import static io.restassured.RestAssured.given;
+
+@QuarkusTest
+class GarageResourceTest extends AbstractProjectResourceTest {
+
+    static final String BASE_PATH = "/api/v1/projects";
+
+    private static final String PROJECT_JSON = "{ \"title\": \"" + TestData.PROJECT_TITLE + "\" }";
+
+    private static final String PROPERTY_JSON = "{ \"title\": \"" + TestData.PROPERTY_TITLE + "\" }";
+
+    private static final String BUILDING_JSON = "{ \"description\":\"" + TestData.BUILDING_DESCRIPTION + "\"," +
+            " \"livingSpace\":\"" + TestData.APARTMENT_LIVING_SPACE + "\"," +
+            " \"title\":\"" + TestData.BUILDING_TITLE + "\"," +
+            " \"commercialSpace\":\"" + TestData.COMMERCIAL_COMMERCIAL_SPACE + "\"," +
+            " \"usableSpace\":\"" + TestData.BUILDING_USABLE_SPACE + "\"," +
+            " \"heatingSpace\":\"" + TestData.APARTMENT_HEATING_SPACE + "\"," +
+            " \"address\": {" +
+            "     \"street\": \"" + TestData.ADDRESS_STREET + "\"," +
+            "     \"city\": \"" + TestData.ADDRESS_CITY + "\"," +
+            "     \"province\": \"" + TestData.ADDRESS_PROVINCE + "\"," +
+            "     \"zip\": \"" + TestData.ADDRESS_ZIP + "\"," +
+            "     \"country\": \"" + TestData.ADDRESS_COUNTRY + "\"" +
+            " } }";
+
+    private static final String GARAGE_JSON = "{ \"title\": \"" + TestData.GARAGE_TITLE + "\"," +
+            " \"description\": \"" + TestData.GARAGE_DESCRIPTION + "\"," +
+            " \"usableSpace\": " + 12.8f + "," +
+            " \"location\": \"" + TestData.GARAGE_LOCATION + "\" }";
+
+    @BeforeEach
+    protected void setupTestUsers() {
+        super.setupTestUsers();
+    }
+
+    @Test
+    void getGarage_FAILED_garageDoesNotExist() {
+        final String projectId = given()
```

Comment:
A lot of repeated steps for creating a project → property → building → then the garage in each test method. Consider extracting common setup logic into helper method, to reduce code redundancy